src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1795,6 +1836,9 @@ def dh_parameters_supported(self, p, g):
 
         return codes[0] == 0
 
+    def dhx_serialization_supported(self):
```

Comment:
DHX is an OpenSSL-ism. We need a more generic concept for what this is for other backends that may have similar limitations. How does the serialization differ?